packages/rest/src/lib/handlers/SequentialHandler.ts

```diff
@@ -209,9 +210,11 @@ export class SequentialHandler implements IHandler {
 			let delay: Promise<void>;
 
 			if (isGlobal) {
+				const offset = normalizeRateLimitOffset(this.manager.options.offset, routeId.bucketRoute);
+
 				// Set RateLimitData based on the global limit
 				limit = this.manager.options.globalRequestsPerSecond;
-				timeout = this.manager.globalReset + this.manager.options.offset - Date.now();
+				timeout = offset - Date.now();
```

Comment:
```suggestion
				timeout = this.manager.globalReset + offset - Date.now();
```

You removed a bit too much. 👋 Global ratelimit